tests/run_tests.py

```diff
@@ -57,21 +136,60 @@
     if args.jenkins:
         os.environ['JENKINS_HOME'] = ''
 
+    # read the environmental variables for backwards compatibility
+    if 'JENKINS_HOME' in os.environ:
+        args.jenkins = True
+    if 'SKIP_SLOW' in os.environ:
+        args.fast = True
+    if 'FEELINGLUCKY' in os.environ:
+        args.lucky = True
+
     sys.path.append(testdir)
     sys.path.append(projdir)
     sys.path.append(os.path.join(projdir, 'src/python'))
 
     start_time = datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")
 
+    loader = unittest.defaultTestLoader
```

Comment:
So we don't want to create an instance of the loader here?

Reply:
`unittest.defaultTestLoader` [provides an instance](https://docs.python.org/3/library/unittest.html#unittest.TestLoader) and we don't need a "custom" one.